web/src/main/webResources/WEB-INF/config-spring-geonetwork.xml

```diff
@@ -229,13 +225,7 @@
 
     <bean class="org.fao.geonet.kernel.datamanager.draft.DraftMetadataIndexer"/>
     <bean class="org.fao.geonet.kernel.datamanager.draft.DraftMetadataManager"/>
-    <bean class="org.fao.geonet.kernel.datamanager.draft.DraftMetadataUtils">
```

Comment:
Keep this, it comes from https://github.com/geonetwork/core-geonetwork/pull/7104